### PR TITLE
Fhdds 774

### PR DESCRIPTION
**Pull request by co-category**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 1]
```diff
@@ -1,44 +1,24 @@
...
+    <li>how to contact HMRC about your application</li>
+</ul>
+</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">Thank you for your help and feedback,</p>
-    <p style="margin: 0 0 30px; font-size: 19px;">Karina Immanuel</p>
-    <p style="margin: 0 0 30px; font-size: 19px;">Digital Service Manager</p>
-    <p style="margin: 0 0 30px; font-size: 19px;">Fulfilment House Due Diligence Scheme</p>
+<p style="margin: 0 0 30px; font-size: 19px;">From the HMRC Fulfilment House Due Diligence Scheme</p>
+<p style="margin: 0 0 30px; font-size: 19px;">Do not reply to this email.</p>
```
Comment (tev1622) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: Not required, already in the email footer

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt [hunk 2]
```diff
@@ -1,48 +1,23 @@
...
-After that we'll make and test a second version of the service based on the feedback we get.
-
-Why you got this email
-This email is being sent to people who've been involved in our research so far, or who've volunteered to help us test this service.
-
-Thank you for your help and feedback,
-Karina Immanuel
-Digital Service Manager
-Fulfilment House Due Diligence Scheme
+Do not reply to this email.
```
Comment (tev1622) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt]: ditto

**Comment by tev1622** [comment]
Your email needs a salutation at the start either with the users name or "Dear Customer"

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt (hunk 2 above)
Comment (co-category) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt]: I took this out from the text version and rendered the email. The email footer is not displayed. It does for the HTML version but not for the TEXT version.

Comment (drainhmrc) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt]: The `@{uk.gov.hmrc.hmrcemailrenderer.templates.helpers.txt.template_footer()}` should render the footer

**Pull request merged**